raybot/commands/whatdidiorder.rb

```diff
@@ -4,34 +4,52 @@
 module RayBot
   module Commands
     class WhatDidIOrder < SlackRubyBot::Commands::Base
-      match (/^.*(whatdidiorder).*$/) do |client, data, match|
+      match (/^.*([wW]hat\s+.*did\s+.*[iI]\s+.*order).*$/) do |client, data, match|
```

Comment:
What is the point of this now? Now `whatdidiorder` doesnt work....

Reply:
fixed